tests/bsim/bluetooth/host/privacy/peripheral/src/tester.c

```diff
@@ -11,53 +11,72 @@
 #include <zephyr/bluetooth/bluetooth.h>
 
 #define EXPECTED_NUM_ROTATIONS 5
+#define MANUFACTURER_DATA_LEN  2
 
-static bt_addr_le_t old_addr;
-static int64_t old_time;
-static int rpa_rotations;
+struct adv_set_data_t {
+	bt_addr_le_t	old_addr;
+	int64_t	old_time;
+	int	rpa_rotations;
```

Comment:
Still some odd indentation here. Have you configured your editor to use 8 spaces as tabs?

Reply:
corrected.